src/components/NavContainer.tsx

```diff
@@ -26,25 +27,33 @@ const NavRoutesToIcon = {
 };
 
 export default function NavFlow() {
+  const { setQuestionState } = useContext(QuestionContext);
+  useEffect(() => {
+    const getQuestions = async () => {
+      const result: any = await API.graphql({
+        query: customListQuestions,
+      });
+      setQuestionState({ items: result.data.listQuestions.items });
+    };
```

Comment:
nit: type `result` like above

Reply:
Since this is a custom query, I don't even have an AWS type for it.